src/Compilers/Core/Portable/AdditionalTextFile.cs

```diff
@@ -42,7 +42,7 @@ public AdditionalTextFile(CommandLineSourceFile sourceFile, CommonCompiler compi
         /// Returns a <see cref="SourceText"/> with the contents of this file, or <c>null</c> if
         /// there were errors reading the file.
         /// </summary>
-        public override SourceText GetText(CancellationToken cancellationToken = default(CancellationToken))
+        public override SourceText GetText(CancellationToken cancellationToken = default)
```

Comment:
:bulb: To minimize the number of users broken by this change prior to 15.3 release, constrain the visibility of 7.1 syntax to a method body, not a signature.